treenode.go

```diff
@@ -646,6 +646,20 @@ func (n *TreeNodeInstance) SendToChildren(msg interface{}) error {
 // If the underlying node is a leaf node this function does
 // nothing.
 func (n *TreeNodeInstance) SendToChildrenInParallel(msg interface{}) error {
+	return collectErrors("Error while sending to %s: %s\n", n.sendToChildrenInParallel(msg))
+}
+
+// SendToChildrenInParallelIgnoreErrs is the same as SendToChildrenInParallel,
+// except it ignores errors if there are fewer than `t` of them.
+func (n *TreeNodeInstance) SendToChildrenInParallelIgnoreErrs(msg interface{}, t int) error {
```

Comment:
SendToChildrenInParallel is called in so few places, maybe it would be better for you to just refactor it now to return []error. Then where you need to check if len(errors < X) you can.

Reply:
The callers are:
./cothority/bftcosi/bftcosi.go:	return bft.SendToChildrenInParallel(&ann)
./cothority/bftcosi/bftcosi.go:	return bft.SendToChildrenInParallel(&ch)
./cothority/bftcosi/bftcosi.go:	return bft.SendToChildrenInParallel(&ch)

Reply:
Fixed now